probe-rs/targets/esp32.yaml

```diff
@@ -2,6 +2,7 @@ name: esp32
 manufacturer:
   id: 0x12
   cc: 0xc
+# chip_detection deliberately missing to avoid misdetecting 1.8V variants
```

Comment:
Can we not detect them at all or not atm?

Reply:
I'm not hopeful for ESP32

Reply:
That being said: we can detect ESP32 overall, just not which VDDIO variant we're working with. I might be overblowing the importance of this (based on the actual risks and overall rarity of the 1.8V variant), but I want to err on the side of caution.